cmd/geth/testdata/config.toml

```diff
@@ -3,7 +3,6 @@ NetworkId = 714
 SyncMode = "full"
 NoPruning = false
 NoPrefetch = false
-LightPeers = 100
```

Comment:
@zzzckck 
please remove this field in https://github.com/bnb-chain/bsc/releases/download/v1.4.15/mainnet.zip
and https://github.com/bnb-chain/bsc/releases/download/v1.4.15/testnet.zip